couchdbkit/client.py

```diff
@@ -716,7 +726,10 @@ def delete_doc(self, doc, **params):
             couch_doc['_rev'] = doc1['_rev']
         elif isinstance(doc1, six.string_types): # we get a docid
             couch_doc = Document(self.cloudant_database, doc1)
-            couch_doc['_rev'] = self.get_rev(doc1)
+            try:
+                couch_doc['_rev'] = self.get_rev(doc1)
+            except ResourceNotFound:
```

Comment:
If `get_rev` is using requests I think it would return `HttpError` right?

Reply:
It catches a 404 error and raises a ```ResourceNotFound``` error instead